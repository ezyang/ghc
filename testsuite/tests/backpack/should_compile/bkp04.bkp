unit p where
    signature A(T) where
        data T
    signature B(T) where
        data T
    signature A(T) where
        import B(T)
