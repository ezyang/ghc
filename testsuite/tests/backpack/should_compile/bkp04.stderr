==== Package p ====
Shape for p
provides:
requires: A -> THIS:A, THIS:A
               hole:B.T{hole:B.T}
          B -> THIS:B
               T{T}
[1 of 3] Compiling A[abstract sig]  ( p/A.hsig, nothing )
[2 of 3] Compiling B[abstract sig]  ( p/B.hsig, nothing )
[3 of 3] Compiling A[abstract sig]  ( p/A.hsig, nothing )
